src/libs/actions/BankAccounts.ts

```diff
@@ -208,6 +208,7 @@ function addPersonalBankAccount(account: PlaidBankAccount) {
             },
         ],
         failureData: [
+            // @ts-expect-error: ONYXKEYS.PERSONAL_BANK_ACCOUNT is conflicting with ONYXKEYS.FORMS.PERSONAL_BANK_ACCOUNT
             {
                 onyxMethod: Onyx.METHOD.MERGE,
                 key: ONYXKEYS.PERSONAL_BANK_ACCOUNT,
```

Comment:
This also conflicts because of the `errors` property. It expects to be FormValue | null | undefined, but the error type is OnyxCommon.Errors.
https://github.com/Expensify/App/blob/bf1ec5adbd63185e95027292793d0622da214128/src/types/form/Form.ts#L4-L16

If we don't pass anything to TInputs, then the value type is expected to be FormValue. If I pass any value to it, then the type error is gone.
https://github.com/Expensify/App/blob/bf1ec5adbd63185e95027292793d0622da214128/src/types/form/PersonalBankAccountForm.ts#L3

```
type PersonalBankAccountForm = Form<"test">
```